### ISSUE TITLE
mt76 might not work properly even after adding 0608 PCI id for RZ608

### ISSUE THREAD

**Issue by lunzima** [ShadowBlip/rz608-fix#1]
An AMD FAE working in AMD's Shenzhen Office could not connect to the Wifi hotspot in that office properly until upgrading kernel version to 5.17-rc3.
Previously he used a 5.16 kernel with patched mt76 driver which has 0608 id for the RZ608 card.

That forum article is written in Chinese.
https://ngabbs.com/read.php?tid=30118483
"Not so difficult to install Linux driver for RZ608", however that embarrassing FAE from AMD still reports connection failure with tons of funny emoticons.

**Comment by pastaq** [comment]
I personally used RZ608 with kernel 5.16 with this fix. Suspend/reboot issues can be solved by unloading the module with systemd as was done in my aya-neo-fixes package (and is still required, even with kernel 5.18). 

https://github.com/ShadowBlip/aya-neo-fixes/blob/main/mt7921e.shutdown
https://github.com/ShadowBlip/aya-neo-fixes/blob/main/systemd-suspend-mods.conf

Have you failed to get this working on a specific piece of hardware?

**Comment by ixervice** [comment]
Tired it. Still cannot work with Holoiso 20220527 or 20220531.

**Comment by pastaq** [comment]
> Tired it. Still cannot work with Holoiso 20220527 or 20220531.

Can you provide the output of 

```lsmod | grep mt``` from terminal?

**Comment by ixervice** [comment]
> > Tired it. Still cannot work with Holoiso 20220527 or 20220531.
> 
> Can you provide the output of
> 
> `lsmod | grep mt` from terminal?

Shows nothing.

BTW, the holoiso system kernel version stays default, as "5.13.0-valve10.1-2-neptune-dri-02144-g7fffaf925dfb" 

![image](https://user-images.githubusercontent.com/18065701/171774042-e7eb2631-f3a0-4c89-a4ed-1a80e03ae173.png)

**Comment by pastaq** [comment]
Try ```sudo modprobe mt7921e```

**Comment by ixervice** [comment]
![image](https://user-images.githubusercontent.com/18065701/171777818-9147f121-7e28-4dc3-884f-0fe9cd8ab9d5.png)

**Comment by pastaq** [comment]
That driver was included in 5.13. I'll install HoloISO on my spare drive and investigate further.

**Comment by pastaq** [comment]
So it looks like the mt7921e kernel was an optional module in kernel 5.13 that was not defaulted in linux-neptune-dri. You will need to select the core/linux install option for holoiso for this fix to work as the base driver it modifies is only present in that kernel option.

**Comment by pastaq** [comment]
you can update your kernel
```
sudo pacman -Sy linux
sudo mkinitcpio -c /etc/mkinitcpio.conf -g /boot/initramfs-linux.img -k 5.16.2-arch1-1.1
reboot
```

**Comment by ixervice** [comment]
The new kernel works.
But I wonder, the magic is the new kernel(5.16+), not the rz608-fix-git AUR Package.
See this.
https://www.reddit.com/r/ayaneo/comments/uku3uh/guide_to_installing_holoiso_unofficial_steamos_30/

**Comment by ixervice** [comment]
Thx for your reply. 
Hope theVakhovskeIsTaken or Steam OS update the new kernel (5.16+) soon.

**Comment by pastaq** [comment]
> The new kernel works. But I wonder, the magic is the new kernel(5.16+), not the rz608-fix-git AUR Package. See this. https://www.reddit.com/r/ayaneo/comments/uku3uh/guide_to_installing_holoiso_unofficial_steamos_30/

This package should fix 5.16.2, preventing the need to alter the repo and install 5.16.5. That being said, HoloISO is still a pain right now and honestly ChimeraOS is a way better option for most users.
